src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/RazorCompletionResolveData.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.CodeAnalysis.Razor.Completion;
+
+internal record RazorCompletionResolveData(
+    [property: JsonPropertyName("TextDocument")] TextDocumentIdentifier TextDocument,
+    [property: JsonPropertyName("data")] object? OriginalData)
```

Comment:
nit: casing doesn't match `TextDocument` and `data`

Reply:
Oh, thank you, this needs a comment. The capital T is vitally important!